handlers/account_missing_warning.py

```diff
@@ -21,15 +20,14 @@ async def fetch_users_missing_pub(conn=conn):
     Fetches all users still missing an public key for reward distribution
     returns array of tuples (user_id, potential_votes)
     """
-
-    leaderboard_rows = fetchLeaderboard(conn, fetch_last_tx(), datetime.now())
+    leaderboard_rows = getUpvoteMap(conn, fetch_last_tx())
     user_rows = fetchUserPubKeys(conn)
     missingUsers = []
 
     for row in leaderboard_rows:
-
-        if not [user for user in user_rows if user[0] == row[0]] and row[0] != client.user.id:
-            print(f"{row[0]} has no pub key connected to their account! They are missing out on {row[1]} upvotes.")
+        if not [user for user in user_rows if user[0] == row] and row != client.user.id:
```

Comment:
This syntax still looks *insane* to me ... but uhh .... I guess that's just 🐍 stuff 🤣

Reply:
Python is fun! The fun is in making the code look insane but readable 🤣